lib/_http_server.js

```diff
@@ -222,6 +222,16 @@ function writeHead(statusCode, reason, obj) {
     headers = obj;
   }
 
+  // RFC 7230 3.3.3.  Message Body Length
+  // Any response to a HEAD request and any response with a 1xx
+  // (Informational), 204 (No Content), or 304 (Not Modified) status
+  // code is always terminated by the first empty line after the
+  // header fields, regardless of the header fields present in the
+  // message, and thus cannot contain a message body.
```

Comment:
This comment doesn't really describe the code below it well. The code doesn't even check for HEAD requests or 1xx/204 responses.